### ISSUE TITLE
failures in us-ar, us-al-shelby

### ISSUE THREAD

**Issue by sbma44** [openaddresses/openaddresses#762]
```
727 sbma44-mbp:~/mapbox/code/js/openaddresses-conform $ node index.js ../../misc/openaddresses/sources/us-ar.json build/
  conform:downloadCache Processing: us-ar +0ms
  conform:downloadCache did not find cached file at build/us-ar.json +2ms
  conform:downloadCache fetching http://www.geostor.arkansas.gov/ArcGIS/rest/services/FEATURE_SERVICES/GEOSTOR_STRUCT/MapServer/0 +1ms
  Downloading [====================] 100% 0.0s

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: Unexpected "h" at position 1 in state START
    at Parser.proto.charError (/Users/tomlee/Dropbox (MapBox)/code/js/openaddresses-conform/node_modules/geojson-stream/node_modules/JSONStream/node_modules/jsonparse/jsonparse.js:84:16)
    at Parser.proto.write (/Users/tomlee/Dropbox (MapBox)/code/js/openaddresses-conform/node_modules/geojson-stream/node_modules/JSONStream/node_modules/jsonparse/jsonparse.js:112:23)
    at Stream.<anonymous> (/Users/tomlee/Dropbox (MapBox)/code/js/openaddresses-conform/node_modules/geojson-stream/node_modules/JSONStream/index.js:27:12)
    at Stream.stream.write (/Users/tomlee/Dropbox (MapBox)/code/js/openaddresses-conform/node_modules/geojson-stream/node_modules/through/index.js:26:11)
    at write (_stream_readable.js:601:24)
    at flow (_stream_readable.js:610:7)
    at ReadStream.pipeOnReadable (_stream_readable.js:642:5)
    at ReadStream.emit (events.js:92:17)
    at emitReadable_ (_stream_readable.js:426:10)
    at emitReadable (_stream_readable.js:422:5)
```

and

```
728 sbma44-mbp:~/mapbox/code/js/openaddresses-conform $ node index.js ../../misc/openaddresses/sources/us-al-shelby.json build/
  conform:downloadCache Processing: us-al-shelby +0ms
  conform:downloadCache did not find cached file at build/us-al-shelby.json +2ms
  conform:downloadCache fetching http://gis.shelbyal.com/ArcGIS/rest/services/Address-Numbers-Capture/MapServer/0 +0ms
  Downloading [====================] 100% 0.0s

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: Unexpected "!" at position 1 in state START
    at Parser.proto.charError (/Users/tomlee/Dropbox (MapBox)/code/js/openaddresses-conform/node_modules/geojson-stream/node_modules/JSONStream/node_modules/jsonparse/jsonparse.js:84:16)
    at Parser.proto.write (/Users/tomlee/Dropbox (MapBox)/code/js/openaddresses-conform/node_modules/geojson-stream/node_modules/JSONStream/node_modules/jsonparse/jsonparse.js:112:23)
    at Stream.<anonymous> (/Users/tomlee/Dropbox (MapBox)/code/js/openaddresses-conform/node_modules/geojson-stream/node_modules/JSONStream/index.js:27:12)
    at Stream.stream.write (/Users/tomlee/Dropbox (MapBox)/code/js/openaddresses-conform/node_modules/geojson-stream/node_modules/through/index.js:26:11)
    at write (_stream_readable.js:601:24)
    at flow (_stream_readable.js:610:7)
    at ReadStream.pipeOnReadable (_stream_readable.js:642:5)
    at ReadStream.emit (events.js:92:17)
    at emitReadable_ (_stream_readable.js:426:10)
    at emitReadable (_stream_readable.js:422:5)
```

**Comment by geobrando** [comment]
us-ar fixed in SHA: 92a01433c3087869b8d5840173128c807ab854d8

For us-al-shelby.. apparently Shelby County, Alabama seems to have decided to offer data only via a Silverlight map.  No way to really rectify this.

**Comment by migurski** [comment]
Looks okay to me: http://data.openaddresses.io/runs/1425022188.203/us-ar.txt
